bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/PerformanceConfiguration.kt

```diff
@@ -48,6 +49,8 @@ public class PerformanceConfiguration private constructor(public val context: Co
      */
     public var networkRequestCallback: NetworkRequestInstrumentationCallback? = null
 
+    public var tracePropagationUrls: Set<Pattern> = emptySet()
```

Comment:
This should have a mutable default value:

```suggestion
    public var tracePropagationUrls: Collection<Pattern> = HashSet()
```